llvm/CMakePresets.json

```diff
@@ -0,0 +1,89 @@
+
+/*
+   This is the default presets for LLVM, if you want to add your own
+   custom one, create a CMakeUserPresets.json instead.
+*/
+
+{
+  "version": 2,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 20,
+    "patch": 0
+  },
+  "configurePresets": [
+    {
+      "name": "default_win",
+      "hidden": true,
+      "generator": "Ninja",
+      "binaryDir": "${sourceParentDir}/out/${presetName}",
+      "cacheVariables": {
+        "LLVM_ENABLE_PROJECTS": "clang;lld;clang-tools-extra"
+      }
+    },
+    {
+      "name": "Debug_Win_x64_msvc",
+      "inherits": "default_win",
+      "displayName": "Debug Win x64 (MSVC)",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Debug",
+        "LLVM_OPTIMIZED_TABLEGEN": "ON"
+      }
+    },
+    {
+      "name": "Release_Win_x64_msvc",
+      "inherits": "default_win",
+      "displayName": "Release Win x64 (MSVC)",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Release",
+        "LLVM_ENABLE_PDB": "ON",
+        "LLVM_ENABLE_ASSERTIONS": "ON"
```

Comment:
Why are asserts enabled for the release build? It doesn't look that is the case for the clang release config.